packages/execute-driver/README.md

```diff
@@ -0,0 +1,35 @@
+# Appium Execute Driver Script Plugin
+
+TODO
```

Comment:
Are these todos expected to be handled before merge?

Reply:
yeah.